### DISCUSSION TITLE
HeaderAttribute 어떻게 사용하나요?

### DISCUSSION THREAD

**Discussion by mayko1234** [naverz/zepeto-studio-kor#351] [5. Q&A]
안녕하세요, 
https://docs.unity3d.com/ScriptReference/HeaderAttribute.html
HeaderAttribute 어떻게 사용하나요?

**Comment by minjujuu** [comment]
declare 쪽 보니까 지금 제공되는 Attribute는 이 세 가지인 것 같습니다..!
![image](https://user-images.githubusercontent.com/79794414/153317679-ee89e1dc-4eb9-45d8-970a-ab479d82e990.png)

**Reply by mayko1234** [reply]
답변 감사합니다.
index.d.ts파일을 보면, HeaderAttribute class가 선언되어있더라구요. 
이걸 어떻게 쓸 수 있는 방법이 있지 않을까 궁금했어요.
![image](https://user-images.githubusercontent.com/97988822/153318771-3c409aa8-1f23-452c-9503-3a5774e59adf.png)
